tests/RobotFramework/tests/cumulocity/inventory/inventory.json

```diff
@@ -1,6 +1,7 @@
 {
     "name": "TST_dark_knight",
     "type": "RPi5",
+    "manufacturer": "Broadcom",
```

Comment:
Is this values actually used anywhere?

Reply:
Resolved by [c3203d8](https://github.com/thin-edge/thin-edge.io/pull/3742/commits/c3203d8cd9a90f2377db7c4c1a2f039f5caf64c5)